sass/layout/_sidebar-left.scss

```diff
@@ -612,15 +612,21 @@
                 opacity: 0.4;
             }
         }
+
+        &.dragging {
+            border-radius: 4px;
+            box-shadow: 0 4px 6px rgba(0, 0, 0, 0.12);
+            color: var(--sidebar-text);
+
+            .SidebarChannelGroupHeader_groupButton {
+                border-radius: 4px;
+            }
+        }
     }
 
     .SidebarChannelGroup:not(.autoSortedCategory) .draggingOver .SidebarChannelGroupHeader:hover {
         box-shadow: inset -1px 0 0 rgba(var(--sidebar-text-rgb), 0.60), inset 0 -1px 0 rgba(var(--sidebar-text-rgb), 0.6), inset 1px 0 0 rgba(var(--sidebar-text-rgb), 0.6), inset 0 1px 0 rgba(var(--sidebar-text-rgb), 0.6);
         border-radius: 4px;
-
-        .SidebarChannelGroupHeader_groupButton {
-            background-color: inherit;
```

Comment:
I'm not entirely sure why this was added, but removing it makes sure that the issues with the border go away.